src/components/cluster/new/CreateNodePoolsCluster.js

```diff
@@ -574,10 +584,19 @@ CreateNodePoolsCluster.propTypes = {
   selectableReleases: PropTypes.array,
   releases: PropTypes.object,
   activeSortedReleases: PropTypes.array,
+  maxAZ: PropTypes.number,
+  minAZ: PropTypes.number,
+  defaultAZ: PropTypes.number,
 };
 
 function mapStateToProps(state) {
-  let availabilityZones = state.app.info.general.availability_zones.zones;
+  const { availability_zones: AZ } = state.app.info.general;
+  const availabilityZones = AZ.zones;
+  // More than 4 AZs is not allowed by now.
+  const maxAZ = AZ.max > 4 ? 4 : AZ.max;
```

Comment:
Can we extract the `4` number into a constant somewhere, so it'll be easier to find in the future?

Reply:
I really think that this very much needed. Tried to do it, unsuccessfully: https://github.com/giantswarm/happa/pull/775

Feel free to bring this up on the next SIG Frontend sync. I will support you, cause I don't feel comfortable at all doing this

Reply:
I would just extract it at the top of the file, it doesn't need to be global (at least for now)

Reply:
How about `Math.min(4, AZ.max)`?

Reply:
You make me think about it! 
I really love the conciseness of `Math.min` but have chosen instead a more resilient approach instead